tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist =
+    flake8
+    py{34,27}-edge
+    py{34,27}-dj18-cms{33,32}
+    py{34,27}-dj19-cms{33,32}
+
+skip_missing_interpreters=True
+
+
+[testenv]
+deps =
+    -r{toxinidir}/tests/requirements.txt
+    dj18: Django>=1.8,<1.9
+    dj19: Django>=1.9,<1.10
+    edge: django-cms
+    cms32: django-cms>=3.2,<3.3
+    cms33: django-cms>=3.3,<3.4
+commands =
+    {envpython} --version
+    {env:COMMAND:python} setup.py test
+
+
+[flake8]
+max-line-length = 80
```

Comment:
80 is too restrictive, as the cms and django now have 120, I recommend at least 99 or 100